modules/openapi-generator/src/main/resources/typescript/http/servers.mustache

```diff
@@ -26,6 +29,18 @@ export class ServerConfiguration<T extends { [key: string]: string }> implements
         return this.variableConfiguration
     }
 
+    public setHeaders(headers: Headers): void {
```

Comment:
nit: Any thoughts about changing the name?
`getHeaders` return all headers - quite expected
But `setHeaders` does not set all headers, instead it's an additive operation merging new headers with  existing ones

Consequently, it's not possible to remove a header previously added to the server configuration.

I mean - it resembles the existing `setVariables` method, true. Not sure if that interface was well thought through, though (e.g. `getConfiguration` instead of `getVariables` indicates otherwise :) )

Reply:
You are right 
I think It is better to change the function 
@amakhrov

Reply:
```
public setHeaders(headers: Headers): void {
     this.headers = headers;
}
```
What do you think?

Reply:
or we can rename it to mergeHeaders or addHeaders

Reply:
@amakhrov

Reply:
addHeaders sounds reasonable!

Reply:
Done 
@amakhrov